Service/CacheService.php

```diff
@@ -284,7 +284,7 @@ public function getObject(string $id): ?array
 
         // Check if object is in the cache ????
         if ($object = $collection->findOne(['_id'=>$id])) {
-            return $object;
+            return json_decode(json_encode($object), true);
```

Comment:
Whut? Wat doen we hier?

Reply:
We weten niet zeker wat er terug komt vanuit de cache, dat kan een bsonobject, array of een ander object zijn (en die objecten bevatten geen van allen een goede toArray functie), door naar json te encoden en weer te decoden krijgen we een eenduidige array (want alle mogelijkheden encoden wel goed naar json).